### DISCUSSION TITLE
Search Wiki.js full-text search subcategorized

### DISCUSSION THREAD

**Discussion by Tartaros47** [requarks/wiki#6668] [Help / Questions] [answered]
Is it possible too expand the search in Wiki.js in such a way that the full-text search can also be subcategorized - like MS Sharepoint?
The user should be guided to the word they are looking for, not just to the page where it was mentioned. Does anyone have any experience or a solution?

database postgreSQL 

Thx & BR

current search:

![Suche_aktuell](https://github.com/requarks/wiki/assets/142981699/d3d39879-0159-48ed-b01e-f895387f5671)

**Comment by NGPixel** [comment, marked as answer]
This is not possible at the moment, but is being addressed in v3.

**Reply by Tartaros47** [reply]
Thx, has it been announced when V3 will be released? WIKI.js would be the right choice for our company - but only with full-text search.

**Reply by NGPixel** [reply]
@Tartaros47 there's already full text search in v2. This issue is about jumping to a specific section when clicking a search result.

**Reply by Tartaros47** [reply]
I agree with you on that. Do you know when V3 will be released? I would like to continue testing WIKI Js - with the correct data and with the latest version (even if BETA)... BR

**Reply by rrg92** [reply]
Wiki 3 dont have any prevision yet. Take  look here for news: https://beta.js.wiki/blog

**Reply by Utsav-ace** [reply]
Does wiki.js provides search from within content? and fetch pages having matching content?

**Reply by amlloyd** [reply]
> @Tartaros47 there's already full text search in v2. This issue is about jumping to a specific section when clicking a search result.

How to we enable full text search??

**Reply by DarkSide666** [reply]
I also do not see a way how to enable full text search in page content not just in page headers ...
If that's so, then this will be a reason to get rid of wiki.js instantly :(
